lib/src/ruby/ruby.cc

```diff
@@ -18,14 +18,14 @@ static const char load_puppet[] =
 "  $LOAD_PATH << Puppet[:libdir]\n"
 "end\n"
 "Facter.reset\n"
+"Facter.search_external([Puppet[:pluginfactdest]])\n"
```

Comment:
I believe we didn't do it this way to keep it consistent with what Facter 2.x was doing to support `-p`.